src/main/java/com/faforever/client/fx/PlatformService.java

```diff
@@ -154,13 +187,21 @@ public void startFlashingWindow(String windowTitle) {
     User32.INSTANCE.FlashWindowEx(flashwinfo);
   }
 
-
   public void stopFlashingWindow(String windowTitle) {
+    startFlashingWindow(windowTitle, null);
+  }
```

Comment:
is this supposed to be calling stopFlashingWindow?

Reply:
Oh, good catch